staging/src/k8s.io/api/policy/v1beta1/types.go

```diff
@@ -47,7 +47,7 @@ type PodDisruptionBudgetSpec struct {
 // PodDisruptionBudgetStatus represents information about the status of a
 // PodDisruptionBudget. Status may trail the actual state of a system.
 type PodDisruptionBudgetStatus struct {
-	// Most recent generation observed when updating this PDB status. PodDisruptionsAllowed and other
+	// Most recent generation observed when updating this PDB status. DisruptionsAllowed and other
```

Comment:
We should actually use the json name in the comment (`disruptionsAllowed`); I like to use backticks to make it even more obvious and less weird to start the sentence with a lowercase letter.

Reply:
Thanks for the comment. I checked comments for other fields in this file. We kind of use both ways, lowercase first letter (`// allowedUnsafeSysctls`) and uppercase first letter (`// AllowedCSIDrivers is`), but no backticks. 

Given this sentence is not the start of the comment, I am inclined to keep it as-is to make it more in-line with other comments

Reply:
@liggitt we should probably document this more clearly-- looks like it's not covered by https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#naming-conventions, is it covered somewhere else?

Reply:
Given that it's not officially documented I won't block the PR, I guess

(Note that it being wrong in other places isn't a reason I accept as valid for keeping it wrong here :) )

Reply:
hmm, not that I know of. need a godoc section in that doc, and an issue tracking sweeping existing types to fix docs